### ISSUE TITLE
keep history across sessions

### ISSUE THREAD

**Issue by dsuoch-intel** [xoofx/kalk#25]
It would be great to have the option to keep my history (content, commands) across sessions.

**Comment by xoofx** [comment]
Yep, there was a plan to do that... but never thought about actually keeping the content as well... that's a good idea
